### PR TITLE
Update ManageAccessGroups.md

### PR DESCRIPTION
**Pull request by davewichers**
Minor edits and typo fixes.

### CODE REVIEW

File: content/admin/manage_orgs/ManageAccessGroups.md [hunk 1]
```diff
@@ -23,20 +23,20 @@ Go to the **Groups** page in the System Administration interface. From there:
 
 ## Organization Groups
 
-Use an organization group to assign authorized users access to organizations and applications. An Organization Admin can create, edit and delete groups from the Organization Settings page. System administrators (EOP only) can also create [Cross-Organization Groups](admin-manageorgs.html#crossorg) to allow users access to more than one organization.
+Use an organization group to assign authorized users access to organizations and applications. An Organization Admin can create, edit, and delete groups from the Organization Settings page. System administrators (EOP only) can also create [Cross-Organization Groups](admin-manageorgs.html#crossorg) to allow users access to more than one organization.
```
Comment (kaitlynmortimer) [inline content/admin/manage_orgs/ManageAccessGroups.md]: Please remove Oxford comma.

File: content/admin/manage_orgs/ManageAccessGroups.md [hunk 2]
```diff
@@ -23,20 +23,20 @@ Go to the **Groups** page in the System Administration interface. From there:
...
 
-Contrast provides four default groups within each organization. These groups provide access to all applications in the organization with the associated role. That role grants or restricts what the user can do with the application. Those roles are:
+Contrast provides four default groups within each organization. These groups provide access to all applications in the organization with the associated role. That role grants or restricts what the user can do with each application. Those roles are:
 
-* **View:** Allows users to see an application's score, libraries and vulnerabilities, and add comments.
+* **View:** Allows users to see an application's score, libraries, and vulnerabilities, and add comments.
```
Comment (kaitlynmortimer) [inline content/admin/manage_orgs/ManageAccessGroups.md]: Please see above comment.

File: content/admin/manage_orgs/ManageAccessGroups.md [hunk 3]
```diff
@@ -23,20 +23,20 @@ Go to the **Groups** page in the System Administration interface. From there:
...
 
-A user assigned to an organization can have various roles across applications within that organization; each role grants or restricts what the user can do with the given application. However, the users associated with the organization may only require awareness of one or few of the many applications within the organization. This allows users to focus directly on what matters to them rather than sorting through information that has no bearing on their work. 
+A user assigned to an organization can have various roles across applications within that organization. Each role grants or restricts what the user can do with the given application. However, the users associated with the organization may only require awareness of one or a few of the many applications within the organization. This allows users to focus directly on what matters to them rather than sorting through information that has no bearing on their work.
```
Comment (kaitlynmortimer) [inline content/admin/manage_orgs/ManageAccessGroups.md]: Please restore original punctuation.

**Comment by davewichers** [comment]
What do you have against Oxford commas? They make things far more clear.
Feel free to ignore my pull request, reedit it however you want, and submit
that instead. I don't have time to reedit my suggestions.

-Dave

**Comment by kaitlynmortimer** [comment]
@davewichers I'll make note of the typos that you fixed here, and edit the doc in another branch. Thanks for your help.

**Pull request closed without merging**